language/move-prover/bytecode/src/verification_analysis_v2.rs

```diff
@@ -241,28 +274,31 @@ fn print_fun_id_set(
 /// Print sets and maps computed during verification analysis
 /// TODO: Complete this and write it properly as Display
 #[allow(dead_code)]
-fn print_invariant_analysis_data(global_env: &GlobalEnv, inv_ana_data: &InvariantAnalysisData) {
-    println!("target_invariants <can't print yet>");
-    print_fun_id_set(
+fn debug_print_invariant_analysis_data(
```

Comment:
Perhaps for a later PR, but note that the FunctionTargetProcessor trait now has since a while a method `fn dump_result(&self, f: &mut fmt::Formatter, env: &GlobalEnv,targets: &FunctionTargetsHolder) -> fmt::Result` which you can implement. If you do so the output of this will be included in the output of `--dump-bytecode` and in .exp files in the bytecode crate. This might be a better way to realize this here. See `mono_analysis.rs` for an example.